lib/screens/CourseDetailScreen.dart

```diff
@@ -1,34 +1,30 @@
 import 'package:flutter/material.dart';
+import 'package:flutter_html/flutter_html.dart';
 import 'package:we_find/model/I18NString.dart';
```

Comment:
Currently this is unused, shall we create a separate issue for implementing multilingual support?